examples/problems/200_custom_cost_matrices.ipynb

```diff
@@ -117,7 +117,7 @@
     {
```

Comment:
<p>I wouldn't call it "manually", but "individually for each term"</p><p> </p><p>"Manually" would rather mean you set a cost matrix manually via <code>set_x/y/yx</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>